Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -1085,9 +1116,13 @@ private func isPath(_ path: HTTPRequest.Path) -> HTTPStubsTestBlock {
 
 private extension HTTPStubsResponse {
 
-    static let emptySuccessResponse: HTTPStubsResponse = .init(data: Data(),
-                                                               statusCode: .success,
-                                                               headers: nil)
+    static var emptySuccessResponse: HTTPStubsResponse {
+        // `HTTPStubsResponse` doesn't have value semantics, it's a mutable class!
+        // This creates a new response each time so modifications in one test don't affect others.
+        return .init(data: Data(),
+                     statusCode: .success,
+                     headers: nil)
+    }
```

Comment:
honestly, I think this might be more intuitive as a function.
Otherwise, since the syntax looks exactly the same, it'd be reasonable if you looked only at the callsite to assume that this bug wasn't fixed. 

If you call it like `emptySuccessResponse()`, you expect to be getting a brand new variable each time

Reply:
Agreed 👍🏻